### PR TITLE
[CI logs] Updated Buildbot output

### PR DESCRIPTION
**Pull request by dlobanox**
traceback from Python subprocess does not show;
only errors show in output; full logs available in logs file on share

### CODE REVIEW

File: build_scripts/build_runner.py [hunk 1]
```diff
@@ -179,9 +179,23 @@ def run(self):
...
                 raise
 
+    def _parse_logs(self, stdout):
+        self.log.debug(stdout)
+        output = ""
+
+        substring = ' error ' if platform.system() == 'Windows' else ' error:'
+        for string in stdout.split('\n'):
+            if substring in string:
+                output += '\n' + string
```
Comment (adydychk) [inline build_scripts/build_runner.py]: I think it is better to use list for the variable `output` and .append() each element after that you can:
```
self.log.error("\n".join(output))
```
So you will reduce number of sting instances.

File: build_scripts/build_runner.py [hunk 2]
```diff
@@ -179,9 +179,23 @@ def run(self):
...
+                else:
+                    self.log.error(process_error.stdout)
                 raise
 
+    def _parse_logs(self, stdout):
+        self.log.debug(stdout)
+        output = ""
+
+        substring = ' error ' if platform.system() == 'Windows' else ' error:'
+        for string in stdout.split('\n'):
```
Comment (dimakozlov) [inline build_scripts/build_runner.py]: .splitlines()

File: build_scripts/build_runner.py [hunk 3]
```diff
@@ -179,9 +179,23 @@ def run(self):
...
+            if substring in string:
```
Comment (dimakozlov) [inline build_scripts/build_runner.py]: are you sure that you need all strings with ' error ' substring? Do you have an example of such errors?

File: build_scripts/build_runner.py [hunk 4]
```diff
@@ -771,9 +785,10 @@ def main():
...
         build_config.run_stage(args.stage)
-    except Exception:
+    except Exception as exc:
         set_output_stream('err')
-        log.exception("Exception occurred")
+        if not isinstance(exc, subprocess.CalledProcessError):
+            log.exception("Exception occurred")
         log.error("%sING FAILED", args.stage.name)
         with open(pathlib.Path(args.root_dir) / 'build_state', 'w') as status:
             status.write(json.dumps({'status': "FAIL"}))
```
Comment (dimakozlov) [inline build_scripts/build_runner.py]: pathlib.Path objects have **write_text** method. I would recommend using it.

File: build_scripts/build_runner.py (hunk 1 above)
Comment (dimakozlov) [reply build_scripts/build_runner.py]: +100500

File: build_scripts/build_runner.py (hunk 3 above)
Comment (dlobanox) [reply build_scripts/build_runner.py]: Examples:
mfx_av1_dec_decode.cpp(92): error C2220: warning treated as error - no 'executable' file generated
LINK : fatal error LNK1257: code generation failed

File: build_scripts/build_runner.py (hunk 1 above)
Comment (dlobanox) [reply build_scripts/build_runner.py]: How will we add '\n' if output contains only one element?

Comment (adydychk) [reply build_scripts/build_runner.py]: I think, that you can initialize output with first empty element "": `output = [""]`  
In this case you will have additional `\n` all the time :)

File: build_scripts/build_runner.py (hunk 4 above)
Comment (adydychk) [reply build_scripts/build_runner.py]: @dimakozlov Can you explain me why write_text method is preferable to use? :)

Comment (dlobanox) [reply build_scripts/build_runner.py]: Less code and more understandable )

**Comment by adydychk** [comment]
@azhogov Do you satisfied? :)

File: build_scripts/build_runner.py [hunk 5]
```diff
@@ -179,9 +179,28 @@ def run(self):
...
-                self.log.error(process_error.stdout)
+                if self.stage == Stage.BUILD:
+                    self._parse_logs(process_error.stdout)
+                else:
+                    self.log.error(process_error.stdout)
                 raise
 
+    def _parse_logs(self, stdout):
+        self.log.info(stdout)
+        output = [""]
```
Comment (azhogov) [inline build_scripts/build_runner.py]: Please add a message like:
The errors below were found in the output. See full log for details.

File: build_scripts/build_runner.py [hunk 6]
```diff
@@ -179,9 +179,28 @@ def run(self):
 
                 self.log.debug(completed_process.stdout)
             except subprocess.CalledProcessError as process_error:
-                self.log.error(process_error.stdout)
+                if self.stage == Stage.BUILD:
```
Comment (azhogov) [inline build_scripts/build_runner.py]: It seems a workaround.
Please add TODO: improve error handling

**Pull request merged**